src/components/Inventory/withdraw/flow/WithdrawalBatchesCellRenderer.tsx

```diff
@@ -132,7 +144,9 @@ export default function WithdrawalBatchesCellRenderer(props: RendererProps<Table
       <CellRenderer
         index={index}
         column={column}
-        value={numberFormatter.format(+row.readyQuantityWithdrawn + +row.notReadyQuantityWithdrawn)}
+        value={numberFormatter.format(
+          +row.readyQuantityWithdrawn + +row.notReadyQuantityWithdrawn + +row.germinatingQuantityWithdrawn
```

Comment:
Should validate nurseryV2 enabled before adding germinatingQuantityWidthrawn?

Reply:
I started out that way, but it will be 0 when nurseryV2 is disabled - we don't allow adding to the input. Also tested it.

Reply:
i went ahead and added it anyways 😃

Reply:
Excellent! :)